### PR TITLE
Adding Union Find - Navie approach

### PR DESCRIPTION
**Pull request by travistrle**
Thanks for creating this project. This is my first pull request. Next, I am going to implement two Union Find improvements according to http://en.wikipedia.org/wiki/Disjoint-set_data_structure.

### CODE REVIEW

File: algorithms/data_structure/union_find.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            return x
+        else:
+            return self.find(self.forests[x])
+
+    def is_connected(self, x, y):
+        self.__validate_ele(x)
+        self.__validate_ele(y)
+        if self.find(x) == self.find(y):
+            return True
+        return False
```
Comment (JoaoGFarias) [inline algorithms/data_structure/union_find.py]: Thanks for the pull request, @travistrle 
Could you change this *if*?
You can do just 
```
return self.find(x) == self.find(y)
```

**Comment by travistrle** [comment]
improve one line code for find function
return self.find(x) == self.find(y)

**Comment by travistrle** [comment]
Update oop encapsulation property

**Comment by travistrle** [comment]
done

**Pull request merged**